src/pages-and-resources/discussions/app-config-form/apps/legacy/LegacyConfigForm.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable func-names */
```

Comment:
why this is needed?

Reply:
Because un name lint errors were failing the build.

Reply:
can we move this after the link where we want to silent the error?

Reply:
Yes.